src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -73,6 +75,11 @@ public void testDateRange() throws IOException, URISyntaxException, ParseExcepti
         verifyAllJson(actualFiles, FT_TEMP_DIR);
     }
 
+    @After
+    public void afterEach() {
+        ErrorSummary.getInstance().clearErrorList();
```

Comment:
maybe make it a before each and place it directly after `setUp()` to make it neater.